overlord/devicestate/handlers_install.go

```diff
@@ -320,5 +320,16 @@ func checkEncryption(model *asserts.Model) (res bool, err error) {
 		return false, nil
 	}
 
```

Comment:
we need to also expand the check above about the "secured" flag to be as well about any case 
where StorageSafety is set to Encrypted

Reply:
similarly the recent  code in snap-bootstrap:

https://github.com/snapcore/snapd/blob/master/cmd/snap-bootstrap/cmd_initramfs_mounts.go#L693

the bit about Secured should become about any model that set storage-safety to encrypted